test/index_if.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright Louis Dionne 2013-2017
```

Comment:
Fuckin' A! I love tests that go through the trouble of testing infinite sequences, thanks for that.